src/hotspot/share/jvmci/vmStructs_jvmci.cpp

```diff
@@ -749,22 +749,11 @@
   static_field(VM_Version, _zva_length, int)                            \
   volatile_nonstatic_field(JavaFrameAnchor, _last_Java_fp, intptr_t*)
 
-#define VM_INT_CONSTANTS_CPU(declare_constant, declare_preprocessor_constant, declare_c1_constant, declare_c2_constant, declare_c2_preprocessor_constant) \
-  declare_constant(VM_Version::CPU_FP)                  \
-  declare_constant(VM_Version::CPU_ASIMD)               \
-  declare_constant(VM_Version::CPU_EVTSTRM)             \
-  declare_constant(VM_Version::CPU_AES)                 \
-  declare_constant(VM_Version::CPU_PMULL)               \
-  declare_constant(VM_Version::CPU_SHA1)                \
-  declare_constant(VM_Version::CPU_SHA2)                \
-  declare_constant(VM_Version::CPU_CRC32)               \
-  declare_constant(VM_Version::CPU_LSE)                 \
-  declare_constant(VM_Version::CPU_STXR_PREFETCH)       \
-  declare_constant(VM_Version::CPU_A53MAC)
+#define DECLARE_INT_CPU_FEATURE_CONSTANT(id, name, bit) GENERATE_VM_INT_CONSTANT_ENTRY(VM_Version::CPU_##id)
+#define VM_INT_CPU_FEATURE_CONSTANTS CPU_FEATURE_FLAGS(DECLARE_INT_CPU_FEATURE_CONSTANT)
```

Comment:
Missing `#undef DECLARE_INT_CPU_FEATURE_CONSTANT`.

Reply:
No, it must stay defined up to the point `VM_INT_CPU_FEATURE_CONSTANTS` is used. Since this is a `.cpp` file, it's ok to leave it defined.

Reply:
I see.